<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextDialog</class>
 <widget class="QDialog" name="TextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>47</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBlod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>bold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxItalics">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>â€Žitalics</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
